samples/Kconfig

```diff
@@ -24,6 +24,13 @@ config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
 	  Add the empty_app_core as a child image for the given sample.
 	  Used by samples that only require SOC_NRF5340_CPUNET.
 
+# nRF Connect SDK might place one or more images between MCUBoot and the
```

Comment:
> between MCUBoot and the application

The default indicates that BOOTLOADER_MCUBOOT is disabled. What am I missing here?

Reply:
It is disabled for SPM, but enabled for the parent app.